dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1399,16 +1396,35 @@ a.download-video {
     vertical-align: middle;
   }
   #labelusername {
-    vertical-align:top;
-    padding-top:7px
+    vertical-align: top;
+    padding-top: 7px;
   }
   #usernamefield {
-    vertical-align:top
+    vertical-align: top;
+  }
+  #user_type-block select, #user_user_age, #gender-block select, #schoolcountry-block select {
+    width: 234px;
   }
   #usernamemessage, #password_message, #password_message_confirmation {
-    display:inline-block;
-    max-width:180px;
-    vertical-align:top
+    display: inline-block;
+    max-width: 180px;
+    vertical-align: top
+  }
+  #password_confirmation-block {
```

Comment:
Agreed that it's goofy, but this is kind of a special case so that we can use Ruby-style naming for the fields (e.g. password_confirmation) but we prefer using - for CSS field names.

I guess we could transform the _ to - but then we end up with more code.